File: public/EasterEgg.py
# Sign-in function
def sign_in():
    username = input("Enter your username: ")
    password = input("Enter your password: ")

    # Save the username and password in a database or file
    # ...

    print("Sign-in successful!")

# Login function
def login():
    username = input("Enter your username: ")
    password = input("Enter your password: ")

    # Retrieve the username and password from the database or file
    # ...

    # Check if the entered username and password match the stored values
    if username == stored_username and password == stored_password:
        print("Login successful!")
    else:
        print("Invalid username or password.")

# Main program
choice = input("Enter '1' to sign in or '2' to login: ")

if choice == '1':
    sign_in()
elif choice == '2':
    login()
else:
    print("Invalid choice.")

File: public/EasterEgg.php
<?php
// FILEPATH: /home/anthony.reis/Área de Trabalho/3-Trimestre_DevWeb/public/index.php

// Sign-in functionality
if ($_SERVER['REQUEST_METHOD'] === 'POST' && isset($_POST['sign_in'])) {
    $username = $_POST['username'];
    $password = $_POST['password'];

    // Validate username and password
    if ($username === 'admin' && $password === 'password') {
        // Successful sign-in
        echo 'Sign-in successful!';
    } else {
        // Invalid credentials
        echo 'Invalid username or password';
    }
}

// Login functionality
if ($_SERVER['REQUEST_METHOD'] === 'POST' && isset($_POST['login'])) {
    $username = $_POST['username'];
    $password = $_POST['password'];

    // Validate username and password
    if ($username === 'admin' && $password === 'password') {
        // Successful login
        echo 'Login successful!';
    } else {
        // Invalid credentials
        echo 'Invalid username or password';
    }
}
?>
